src/excalidraw-app/index.tsx

```diff
@@ -575,6 +600,14 @@ const ExcalidrawWrapper = () => {
           },
           files,
         );
+
+        if (error) {
+          setErrorMessage(`${error}`); //TODO
+        }
```

Comment:
this thing will show the error twice (inside the exportToBackend as window.alert() and as an error modal afterwards). You're also stringifying the Error object. I suppose that's what the TODO was for.

In any case, I've fixed it on my end (and also moved the error modal inside the <Excalidraw/> component which was an oversight from a previous PR by me).